src/utils/common_utils.c

```diff
@@ -0,0 +1 @@
+#include "common_utils.h"
```

Comment:
Once again, do we really want to introduce an extra translation unit and add to total link time?